### PR TITLE
DP-747-Add-JdbcConnection-examples-to-Github

### PR DESCRIPTION
**Pull request by Bryan-Noel**

### CODE REVIEW

File: src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+        JdbcConnection connection = new JdbcConnection()
+                .setDriverClassName("org.postgresql.Driver")
+                .setUrl("jdbc:postgresql://localhost:5432/customers")
+                .setUsername("username")
+                .setPlainTextPassword("password");
+
+        for(JdbcTable table : connection.loadTables().getTables()) {
+
+            Catalog catalog = new Catalog();
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java]: It looks like we're creating a catalog for each table (above) and adding one table to each catalog's table (below).  Is this the intent?  

Let's discuss more tomorrow.  The JavaBeans we need to create are like JPA -- one per table -- not replicating what's already in the JdbcXXX classes.  Sorry for the misunderstanding.

**Review by DeleTaylor — CHANGES_REQUESTED** [review]

File: src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java (hunk 1 above)
Comment (Bryan-Noel) [reply src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java]: updated the PR

File: src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+        System.out.println(code.getSource());
+    }
+
+    public static void createTableClass(JdbcTable table) {
+        sourceWriter.println("public class %s {", table.getName());
+        sourceWriter.indent();
+        sourceWriter.println();
+        for(JdbcTableColumn column : table.getColumns()) {
+            sourceWriter.println("private %s %s;", column.getMethodSuffix(), column.getName());
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java:36]: @Bryan-Noel is `column.getMethodSuffix()` the right thing to use here?  Does it handle date and time columns well?

**Review by DeleTaylor — APPROVED** [review]

File: src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java (hunk 2 above)
Comment (Bryan-Noel) [reply src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java:36]: @DeleTaylor , just tested this with Date, Time and Timestamp columns and it does handle those types well.

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**